packages/client/components/PokerSidebarEstimateSection.tsx

```diff
@@ -78,19 +78,20 @@ const PokerSidebarEstimateSection = (props: Props) => {
 
     let sortOrder
     if (destination.index === 0) {
-      sortOrder = destinationTopic.sortOrder - SORT_STEP + dndNoise()
+      sortOrder = destinationTopic.sortOrderFirstDimension - SORT_STEP + dndNoise()
     } else if (destination.index === stageSummaries!.length - 1) {
-      sortOrder = destinationTopic.sortOrder + SORT_STEP + dndNoise()
+      sortOrder = destinationTopic.sortOrderLastDimension + SORT_STEP + dndNoise()
     } else {
       const offset = source.index > destination.index ? -1 : 1
       sortOrder =
-        (stageSummaries[destination.index + offset]!.sortOrder + destinationTopic.sortOrder) / 2 +
+        (stageSummaries[destination.index + offset]!.sortOrderLastDimension + destinationTopic.sortOrderFirstDimension) / 2 +
         dndNoise()
```

Comment:
I need to do this because the old way would break when number of dimensions > 2.